crates/wit/tests/generation_tests/exports/arrays/expanded.rs

```diff
@@ -55,7 +48,7 @@ pub unsafe fn __fce_generated_wrapper_func_inner_arrays_1(arg_0: u32, arg_1: u32
             }
             result
         }
-        let mut arg: Vec<u64> = Vec::from_raw_parts(offset as _, size as _, size as _);
+        let mut arg: Vec<u32> = Vec::from_raw_parts(offset as _, (2 * size) as _, (2 * size) as _);
```

Comment:
how do you feel about making them (2,4) constants?

Reply:
It's an autogenerated code, but anyway I added consts.